components/create_swagger_config.py

```diff
@@ -1202,10 +1213,6 @@ def resolve_schema_issues(definition_name, isi_schema,
         # Issue #10: Update required field to draft 4 style
         elif (definition_name.startswith('Job') and 'items' in prop and
               'required' in prop['items']):
-            if prop['items']['required']:
-                if (is_response_object is False or
-                        not isinstance(prop['type'], list)):
-                    required_props.append(prop_name)
```

Comment:
It looks like we're simply dropping the draft 3 style required attribute from fields without updating it to draft 4?

Reply:
Yes, I should actually update the warning message. I was originally attempting to convert from draft 3 to draft 4, but it turns out that the properties were not actually required. The usage of the `required` attribute was invalid according to JSON schema, so I am now just removing it to align with the actual implementation.